app/brave_settings_strings.grdp

```diff
@@ -1378,6 +1378,15 @@ Leo. This action can't be undone.
   <message name="IDS_SETTINGS_LEO_ASSISTANT_INPUT_MODEL_API_KEY_TOOLTIP_INFO" desc="A brief description for the input title for adding API key">
     Some serving frameworks may require authentication credentials, such as an API key or an access token. These will be added to the request header.
   </message>
+  <message name="IDS_SETTINGS_LEO_ASSISTANT_INPUT_CONTEXT_SIZE" desc="An input title for context size.">
```

Comment:
Nit. Maybe this `desc` value and the others below could be a bit more descriptive for someone who's translating this text but has no idea what size here refers. Is the context size a number of characters? The size of a windows?

Reply:
Tried to find a good balance with `[desc]` attributes in more recent commits. Thank you for the feedback!